modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/accordionList/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
+ Everyone is permitted to copy and distribute verbatim copies
+ of this license document, but changing it is not allowed.
```

Comment:
Are we adding two plugins just for the test? This seems massively more then necessary. And adding licences seems wrong, this may have serious legal and product consequences. If we wanted this functionality in a form of a new plugin, there would need to be a product consideration first. Then, if we do want it, we would likely want to develop our own plugin, inspired by the existing one. cc @dsanz @herrardo 

Instead of directly checking the case in the LPP, it might make sense to check if removal of code causes regression to the original issue. In a PW test, we can replicate steps in [LPS-191427 Drag and Drop not working correctly for image in CKEditor](https://liferay.atlassian.net/browse/LPS-191427). If the issue does not repeat, if drag-and-drop works as expected, that means test sufficiently covers default drag-and-drop case. And that covers custom plugins that rely on default behavior, like that one in LPP.

Reply:
Hello @markocikos,
Yeah you're suggestion makes sense and would be easier to move forward with. I'll get on that.
Thank you for taking a look at this!